onset_maproom/layout.py

```diff
@@ -386,7 +386,15 @@ def map_layout():
                         id="layers_control",
                     ),
                     dlf.LayerGroup(id="layers_group"),
-                    dlf.ScaleControl(imperial=False, position="bottomleft"),
+                    dlf.ScaleControl(imperial=False, position="topright"),
+                    dlf.Colorbar(
+                        id="colorbar",
+                        min=0,
+                        position="bottomleft",
+                        width=300,
+                        height=10,
+                        opacity=.8,
```

Comment:
Right, I don't actually want it to be transparent, but I read a recommendation to use the same opacity as the data so the colors match. Now that I think about it, even that doesn't really ensure that the colors will match, because the background layer isn't a constant color. Agreed that drawing important features on top of the data is the best way to do it. Using transparency is just a cheap hack.

Reply:
Got it. It may make sense to put a white background behind the colorscale so that it is not "polluted" by what's behind, and will also insure readability of ticks labels. And that would apply in both set up: transparency hack and features on top.

Reply:
Agreed about putting a white background under the colorscale.